### ISSUE TITLE
cityscapes sis pretrained model

### ISSUE THREAD

**Issue by aarjunsrinivasan** [EndlessSora/TSIT#9]
I am getting an error when I run test script for cityscapes sis

The loaded model is missing keys

Missing key(s) in state_dict: "style_stream.res_0.conv_0.weight_orig", "style_stream.res_0.conv_0.weight", "style_stream.res_0.conv_0.weight_u", "style_stream.res_0.conv_0.bias", "style_stream.res_0.conv_0.weight_orig", "style_stream.res_0.conv_0.weight_u", "style_stream.res_0.conv_0.weight_v", "style_stream.res_0.conv_1.weight_orig", "style_stream.res_0.conv_1.weight", "style_stream.res_0.conv_1.weight_u", "style_stream.res_0.conv_1.bias", "style_stream.res_0.conv_1.weight_orig", "style_stream.res_0.conv_1.weight_u", "style_stream.res_0.conv_1.weight_v", "style_stream.res_0.conv_s.weight_orig", "style_stream.res_0.conv_s.weight", "style_stream.res_0.conv_s.weight_u", "style_stream.res_0.conv_s.weight_orig", "style_stream.res_0.conv_s.weight_u", "style_stream.res_0.conv_s.weight_v", "style_stream.res_1.conv_0.weight_orig", "style_stream.res_1.conv_0.weight", "style_stream.res_1.conv_0.weight_u", "style_stream.res_1.conv_0.bias", "style_stream.res_1.conv_0.weight_orig", "style_stream.res_1.conv_0.weight_u", "style_stream.res_1.conv_0.weight_v", "style_stream.res_1.conv_1.weight_orig", "style_stream.res_1.conv_1.weight", "style_stream.res_1.conv_1.weight_u", "style_stream.res_1.conv_1.bias", "style_stream.res_1.conv_1.weight_orig", "style_stream.res_1.conv_1.weight_u", "style_stream.res_1.conv_1.weight_v", "style_stream.res_1.conv_s.weight_orig", "style_stream.res_1.conv_s.weight", "style_stream.res_1.conv_s.weight_u", "style_stream.res_1.conv_s.weight_orig", "style_stream.res_1.conv_s.weight_u", "style_stream.res_1.conv_s.weight_v", "style_stream.res_2.conv_0.weight_orig", "style_stream.res_2.conv_0.weight", "style_stream.res_2.conv_0.weight_u", "style_stream.res_2.conv_0.bias", "style_stream.res_2.conv_0.weight_orig", "style_stream.res_2.conv_0.weight_u", "style_stream.res_2.conv_0.weight_v", "style_stream.res_2.conv_1.weight_orig", "style_stream.res_2.conv_1.weight", "style_stream.res_2.conv_1.weight_u", "style_stream.res_2.conv_1.bias", "style_stream.res_2.conv_1.weight_orig", "style_stream.res_2.conv_1.weight_u", "style_stream.res_2.conv_1.weight_v", "style_stream.res_2.conv_s.weight_orig", "style_stream.res_2.conv_s.weight", "style_stream.res_2.conv_s.weight_u", "style_stream.res_2.conv_s.weight_orig", "style_stream.res_2.conv_s.weight_u", "style_stream.res_2.conv_s.weight_v", "style_stream.res_3.conv_0.weight_orig", "style_stream.res_3.conv_0.weight", "style_stream.res_3.conv_0.weight_u", "style_stream.res_3.conv_0.bias", "style_stream.res_3.conv_0.weight_orig", "style_stream.res_3.conv_0.weight_u", "style_stream.res_3.conv_0.weight_v", "style_stream.res_3.conv_1.weight_orig", "style_stream.res_3.conv_1.weight", "style_stream.res_3.conv_1.weight_u", "style_stream.res_3.conv_1.bias", "style_stream.res_3.conv_1.weight_orig", "style_stream.res_3.conv_1.weight_u", "style_stream.res_3.conv_1.weight_v", "style_stream.res_3.conv_s.weight_orig", "style_stream.res_3.conv_s.weight", "style_stream.res_3.conv_s.weight_u", "style_stream.res_3.conv_s.weight_orig", "style_stream.res_3.conv_s.weight_u", "style_stream.res_3.conv_s.weight_v", "style_stream.res_4.conv_0.weight_orig", "style_stream.res_4.conv_0.weight", "style_stream.res_4.conv_0.weight_u", "style_stream.res_4.conv_0.bias", "style_stream.res_4.conv_0.weight_orig", "style_stream.res_4.conv_0.weight_u", "style_stream.res_4.conv_0.weight_v", "style_stream.res_4.conv_1.weight_orig", "style_stream.res_4.conv_1.weight", "style_stream.res_4.conv_1.weight_u", "style_stream.res_4.conv_1.bias", "style_stream.res_4.conv_1.weight_orig", "style_stream.res_4.conv_1.weight_u", "style_stream.res_4.conv_1.weight_v", "style_stream.res_4.conv_s.weight_orig", "style_stream.res_4.conv_s.weight", "style_stream.res_4.conv_s.weight_u", "style_stream.res_4.conv_s.weight_orig", "style_stream.res_4.conv_s.weight_u", "style_stream.res_4.conv_s.weight_v", "style_stream.res_5.conv_0.weight_orig", "style_stream.res_5.conv_0.weight", "style_stream.res_5.conv_0.weight_u", "style_stream.res_5.conv_0.bias", "style_stream.res_5.conv_0.weight_orig", "style_stream.res_5.conv_0.weight_u", "style_stream.res_5.conv_0.weight_v", "style_stream.res_5.conv_1.weight_orig", "style_stream.res_5.conv_1.weight", "style_stream.res_5.conv_1.weight_u", "style_stream.res_5.conv_1.bias", "style_stream.res_5.conv_1.weight_orig", "style_stream.res_5.conv_1.weight_u", "style_stream.res_5.conv_1.weight_v", "style_stream.res_6.conv_0.weight_orig", "style_stream.res_6.conv_0.weight", "style_stream.res_6.conv_0.weight_u", "style_stream.res_6.conv_0.bias", "style_stream.res_6.conv_0.weight_orig", "style_stream.res_6.conv_0.weight_u", "style_stream.res_6.conv_0.weight_v", "style_stream.res_6.conv_1.weight_orig", "style_stream.res_6.conv_1.weight", "style_stream.res_6.conv_1.weight_u", "style_stream.res_6.conv_1.bias", "style_stream.res_6.conv_1.weight_orig", "style_stream.res_6.conv_1.weight_u", "style_stream.res_6.conv_1.weight_v", "style_stream.res_7.conv_0.weight_orig", "style_stream.res_7.conv_0.weight", "style_stream.res_7.conv_0.weight_u", "style_stream.res_7.conv_0.bias", "style_stream.res_7.conv_0.weight_orig", "style_stream.res_7.conv_0.weight_u", "style_stream.res_7.conv_0.weight_v", "style_stream.res_7.conv_1.weight_orig", "style_stream.res_7.conv_1.weight", "style_stream.res_7.conv_1.weight_u", "style_stream.res_7.conv_1.bias", "style_stream.res_7.conv_1.weight_orig", "style_stream.res_7.conv_1.weight_u", "style_stream.res_7.conv_1.weight_v".

**Comment by EndlessSora** [comment]
Please specify `--task 'SIS'`.  For reference, use [this test script](https://github.com/EndlessSora/TSIT/blob/master/test_scripts/sis_cityscapes.sh). To show generated images only, remove `--show_input`.

**Comment by aarjunsrinivasan** [comment]
I got it I had to set no_ss to True

**Comment by z6703** [comment]
Are you using paired real images and labels for inference in SIS, like cityscapes ?  Dont you feel it strange to generate a image from itself ?

**Comment by z6703** [comment]
zwf，已收到！
